### DISCUSSION TITLE
Benchmarks

### DISCUSSION THREAD

**Discussion by jordanparker6** [qdrant/qdrant#62] [General]
Do you have any benchmarks that compare performance against similar tooling (e.g. Milvus, Weaviate and FAISS).

**Comment by generall** [comment]
Hi, yes, we have a number of benchmarks and not only search speed https://github.com/qdrant/benchmark !
If you are interested in direct comparison with existing vector search solutions you can look at our search speed benchmark https://github.com/qdrant/benchmark/tree/master/search_benchmark performed on one of the reference datasets used in https://github.com/erikbern/ann-benchmarks 

However, I would not say that the speed bench is the most interesting. Qdrant uses the standard HNSW algorithm, and its performance differs only when filters are used. At the same time, it makes little sense to compare directly C++ libraries and services, because a service operates with other interfaces.

I would also pay attention to the search consistency when using filters. That is is what makes Qdrant different from all the other solutions. In contrast to them, the accuracy of the search is not lost due to the increased integrity of the HNSW graph in Qdrant. Read more about it here https://qdrant.tech/documentation/indexing/#filtrable-index

**Reply by cgr71ii** [reply]
Hi! I've taken a look at the benchmarks repo, but I don't see comparisons with other solutions like FAISS, but only numbers related to `qdrant`. Am I wrong?

Thanks!

**Reply by generall** [reply]
Hi @cgr71ii! Here is comparison with other engines - https://qdrant.tech/benchmarks/

And here is why we think that direct comparison with FAISS and other libraries is incorrect https://qdrant.tech/benchmarks/#why-you-are-not-comparing-with-faiss-or-annoy
